bluebottle/recurring_donations/management/commands/process_monthly_donations.py

```diff
@@ -82,6 +84,13 @@ def handle(self, *args, **options):
         try:
             tenant = Client.objects.get(client_name=options['tenant'])
             connection.set_tenant(tenant)
+
+            # Also load tenant properties
+            tenant_model_path = tenant.__module__
+            tenant_app_path = tenant_model_path.replace('.models', '')
+            properties = importlib.import_module(tenant_app_path).properties
+            properties.set_tenant(tenant)
+
```

Comment:
This can be replaced with the LocalTenant contextmanager. See (bluebottle/project/management/commands/status_realised.py:31)